apps/api/src/entities/occupation.entity.ts

```diff
@@ -0,0 +1,18 @@
+import { Column, Entity, JoinTable, ManyToMany, PrimaryGeneratedColumn } from 'typeorm';
+import { CareActivity } from './careactivity.entity';
+
+@Entity()
+export class Occupation {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ type: 'varchar', length: 255, nullable: false, unique: true })
+  occupationName: string;
+
+  @Column({ type: 'boolean', nullable: false, default: false })
+  isRegulated: boolean;
+
+  @ManyToMany(() => CareActivity, careactivity => careactivity.occupations)
+  @JoinTable()
+  careActivities: CareActivity[];
```

Comment:
If we do this mapping, then how do we know what type of operation is allowed on the care activity for this occupation (the X, A & C(E)). So, this mapping should lie in a separate mapping table